tomviz/DataSource.cxx

```diff
@@ -232,16 +232,26 @@ bool DataSource::appendSlice(vtkImageData* slice)
 
 void DataSource::setFileName(const QString& filename)
 {
-  auto reader = m_json.value("reader").toObject(QJsonObject());
-  reader["fileName"] = filename;
-  m_json["reader"] = reader;
+  // AG: remove fileName and fileNames redundancy. but keep backward
```

Comment:
We can see from git blame who authored comments, no need to prefix with initials etc. Ideally keep as a sentence, capital first letter, full stop/period. I think this should be removed (looks more like the commit message to me), or reworded to explain that it is a convenience method.